src/tests/LevelTests.cs

```diff
@@ -8,11 +8,11 @@ namespace Project.Tests {
     public class LevelTests {
         [Test]
         public void TestRoomValidity() {
-            //Create new level. Constructor randomly generates a set of rooms and connections
-            var level = new Level();
-            
-            //Ensure that all rooms have at least 2 connections so the player can't get stuck
-            foreach (var room in level.Rooms) {
+			//Generate a new level
+			Level level = LevelGenerator.TryGenerateLevel();
+
+			//Ensure that all rooms have at least 2 connections so the player can't get stuck
+			foreach (var room in level.Rooms) {
                 Assert.IsTrue(room.ConnectedRooms.Length >= 2);
             }
         }
```

Comment:
I would add a test where it generates the level about 1000+ times to get a little extra security.

Reply:
Done.